catalog/transformer/catalog-transformer-csv/src/main/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformer.java

```diff
@@ -169,7 +170,9 @@ private void printData(final CSVPrinter csvPrinter, final Iterator iterator) {
   }
 
   private Set<AttributeDescriptor> getAllRequestedAttributes(
-      final List<Result> results, final Set<String> hiddenFields) {
+      final List<Result> results,
+      final Set<String> hiddenFields,
+      final HashSet<String> requestedFields) {
 
```

Comment:
:pencil2: We should pass in a `java.util.Set` here instead of `java.util.HashSet` so we're using the interface instead of an implementation of it.

Reply:
Fixed bcb6500b403ee42e5cc98f26b1ca7839f9430aea